### ISSUE TITLE
Document a workaround: clicking on a brush icon with the right button of the mouse causes an error

### ISSUE THREAD

**Issue by lunlune** [mypaint/mypaint#706]
#### Description of the problem

for an error, i deleted the favourites category of the brushes. now, every time i click with the right "button" of the mouse on a brush icon, i always get an error message, it can be fixed?
#### Basic system details

MyPaint version: 1.2.1-beta.1+gitexport.3198e3f
Operating system: windows 10
Desktop environment: windows?
#### Steps to reproduce

[☞ Tell us how to reproduce the bug in clear, numbered steps.]
1. Start MyPaint
2. delete the favourites category of brushes
3. click on a brush with the right button of the mouse
#### Backtraces or error messages

``````
```python
Mypaint version: 1.2.1-beta.1+gitexport.3198e3f
System information: Windows-10-10.0.10240-SP0
Using: Python 2.7.11, GTK 3.20.3, GdkPixbuf 2.34.0, Cairo 1.15.2, GLib 2.48.0
Traceback (most recent call last):
  File "C:\Program Files\MyPaint\share\mypaint/gui/brushselectionwindow.py", line 123, button_press_cb(self=<brushselectionwindow.BrushList object at 0x0000...selectionwindow+BrushList at 0x000000000a1279c0)>, widget=<brushselectionwindow.BrushList object at 0x0000...selectionwindow+BrushList at 0x000000000a1279c0)>, event=<Gdk.EventButton object at 0x000000000cf56ec0 (void at 0x000000000c3bad80)>)
            self.app.device_monitor.device_used(device)
            return pixbuflist.PixbufList.button_press_cb(self, widget, event)
  variables: {'widget': ('local', <brushselectionwindow.BrushList object at 0x000000000b7f7b90 (gui+brushselectionwindow+BrushList at 0x000000000a1279c0)>), 'self': ('local', <brushselectionwindow.BrushList object at 0x000000000b7f7b90 (gui+brushselectionwindow+BrushList at 0x000000000a1279c0)>), 'pixbuflist.PixbufList.button_press_cb': ('global', <unbound method PixbufList.button_press_cb>), 'event': ('local', <Gdk.EventButton object at 0x000000000cf56ec0 (void at 0x000000000c3bad80)>)}
  File "C:\Program Files\MyPaint\share\mypaint/gui/pixbuflist.py", line 296, button_press_cb(self=<brushselectionwindow.BrushList object at 0x0000...selectionwindow+BrushList at 0x000000000a1279c0)>, widget=<brushselectionwindow.BrushList object at 0x0000...selectionwindow+BrushList at 0x000000000a1279c0)>, event=<Gdk.EventButton object at 0x000000000cf56ec0 (void at 0x000000000c3bad80)>)
                self.set_selected(item)
                self.item_popup(item)
            else:
  variables: {'item': ('local', <ManagedBrush u'deevad/rough' p=None>), 'self.item_popup': ('local', <MethodWithObservers gui.brushselectionwindow.BrushList.item_popup>)}
  File "C:\Program Files\MyPaint\share\mypaint/lib/observable.py", line 284, __call__(self=<MethodWithObservers gui.brushselectionwindow.BrushList.item_popup>, *args=(<ManagedBrush u'deevad/rough' p=None>,), **kwargs={})
                try:
                    observer(observed, *args, **kwargs)
                except BoundObserverMethod._ReferenceError:
  variables: {'observed': ('local', <brushselectionwindow.BrushList object at 0x000000000b7f7b90 (gui+brushselectionwindow+BrushList at 0x000000000a1279c0)>), 'args': ('local', (<ManagedBrush u'deevad/rough' p=None>,)), 'observer': ('local', <BoundObserverMethod gui.brushselectionwindow.BrushList._item_popup_cb>), 'kwargs': ('local', {})}
  File "C:\Program Files\MyPaint\share\mypaint/lib/observable.py", line 467, __call__(self=<BoundObserverMethod gui.brushselectionwindow.BrushList._item_popup_cb>, observed=<brushselectionwindow.BrushList object at 0x0000...selectionwindow+BrushList at 0x000000000a1279c0)>, *args=(<ManagedBrush u'deevad/rough' p=None>,), **kwargs={})
                raise self._ReferenceError
            self._observer_func(observer, observed, *args, **kwargs)
            del observer
  variables: {'observed': ('local', <brushselectionwindow.BrushList object at 0x000000000b7f7b90 (gui+brushselectionwindow+BrushList at 0x000000000a1279c0)>), 'args': ('local', (<ManagedBrush u'deevad/rough' p=None>,)), 'self._observer_func': ('local', <function _item_popup_cb at 0x0000000009dc60c8>), 'observer': ('local', <brushselectionwindow.BrushList object at 0x000000000b7f7b90 (gui+brushselectionwindow+BrushList at 0x000000000a1279c0)>), 'kwargs': ('local', {})}
  File "C:\Program Files\MyPaint\share\mypaint/gui/brushselectionwindow.py", line 166, _item_popup_cb(self=<brushselectionwindow.BrushList object at 0x0000...selectionwindow+BrushList at 0x000000000a1279c0)>, self_=<brushselectionwindow.BrushList object at 0x0000...selectionwindow+BrushList at 0x000000000a1279c0)>, brush=<ManagedBrush u'deevad/rough' p=None>)
            time = gtk.get_current_event_time()
            self.menu = BrushPopupMenu(self, brush)
            self.menu.show_all()
  File "C:\Program Files\MyPaint\share\mypaint/gui/brushselectionwindow.py", line 175, __init__(self=<brushselectionwindow.BrushPopupMenu object at 0...tionwindow+BrushPopupMenu at 0x000000000b84a8d0)>, bl=<brushselectionwindow.BrushList object at 0x0000...selectionwindow+BrushList at 0x000000000a1279c0)>, brush=<ManagedBrush u'deevad/rough' p=None>)
            super(BrushPopupMenu, self).__init__()
            faves = bl.bm.groups[brushmanager.FAVORITES_BRUSH_GROUP]
            if brush not in faves:
  variables: {'bl.bm.groups': ('local', {u'Tone': [<ManagedBrush u'tone/Sketch_Pencil_1' p=None>, <ManagedBrush u'tone/Sketch_Pencil_2' p=None>, <ManagedBrush u'tone/Black_Pencil' p=None>, <ManagedBrush u'tone/Basic_Sketch_Pencil' p=None>, <ManagedBrush u'tone/Sketch_Pencil_Light' p=None>, <ManagedBrush u'tone/Pencil_Shader_1' p=None>, <ManagedBrush u'tone/Basic_Pencil_Shader' p=None>, <ManagedBrush u'tone/Normal_Pencil' p=None>, <ManagedBrush u'tone/Black_Smooth_Pencil' p=None>, <ManagedBrush u'tone/Graphite_Powder' p=None>, <ManagedBrush u'tone/Graphite_Powder_Smudge' p=None>, <ManagedBrush u'tone/Smudge_Pencil_1' p=None>, <ManagedBrush u'tone/Smudge_Pencil_2' p=None>, <ManagedBrush u'tone/Smudge_Pencil_3' p=None>, <ManagedBrush u'tone/Smudge_Pencil_Only_1' p=None>, <ManagedBrush u'tone/Smudge_Pencil_Only_2' p=None>, <ManagedBrush u'tone/Offset_Pencil_aka_Korsi' p=None>, <ManagedBrush u'tone/Conte_Pencil_1' p=None>, <ManagedBrush u'tone/Conte_Pencil_2' p=None>, <ManagedBrush u'tone/Soft_Edge_Eraser' p=None>, <ManagedBrush u'tone/Hard_Edge_Eraser' p=None>, <ManagedBrush u'tone/Tones_Liner' p=None>, <ManagedBrush u'tone/Ink_Filler' p=None>, <ManagedBrush u'tone/Sketch_Ink_2' p=None>, <ManagedBrush u'tone/Sketch_Ink_1' p=None>, <ManagedBrush u'tone/Brush_Pen' p=None>, <ManagedBrush u'tone/Worn_Brush_Pen' p=None>, <ManagedBrush u'tone/G-Pen' p=None>, <ManagedBrush u'tone/Maru_Pen' p=None>, <ManagedBrush u'tone/Smooth_Ink_1' p=None>, <ManagedBrush u'tone/Smooth_Ink_2' p=None>, <ManagedBrush u'tone/Smooth_Light_Ink' p=None>, <ManagedBrush u'tone/Delay_Ink_1' p=None>, <ManagedBrush u'tone/Delay_Ink_2' p=None>, <ManagedBrush u'tone/Wedge_Ink' p=None>, <ManagedBrush u'tone/Tag_Ink' p=None>, <ManagedBrush u'tone/Whirl_Ink' p=None>, <ManagedBrush u'tone/Ink_Splatter' p=None>, <ManagedBrush u'tone/Polka_Dot_Ink' p=None>, <ManagedBrush u'tone/Dry_Brush_Ink' p=None>, <ManagedBrush u'tone/Flat_Dot_Texture_Ink' p=None>, <ManagedBrush u'tone/Flat_Dot_Follow_Ink_1' p=None>, <ManagedBrush u'tone/Flat_Dot_Follow_Ink_2' p=None>, <ManagedBrush u'tone/Velocity_Sketcher_Ink' p=None>, <ManagedBrush u'tone/Wet_Ink_Brush_1' p=None>, <ManagedBrush u'tone/Wet_Ink_Brush_2' p=None>, <ManagedBrush u'tone/Harmony_Sketchy' p=None>, <ManagedBrush u'tone/Sparkle' p=None>, <ManagedBrush u'tone/Streak_Brush' p=None>, <ManagedBrush u'tone/Furry_1' p=None>, <ManagedBrush u'tone/Extreme_Fine_Airbrush' p=None>, <ManagedBrush u'tone/Fine_Airbrush_1' p=None>, <ManagedBrush u'tone/Wet_Airbrush_Spray_1' p=None>, <ManagedBrush u'tone/Wet_Airbrush_Spray_2' p=None>, <ManagedBrush u'tone/Wet_Airbrush_Spray_3' p=None>, <ManagedBrush u'tone/Wet_Airbrush_Spray_4' p=None>, <ManagedBrush u'tone/Ball_Spray_Brush_1' p=None>, <ManagedBrush u'tone/Ball_Spray_Brush_2' p=None>, <ManagedBrush u'tone/Ball_Spray_Brush_3' p=None>, <ManagedBrush u'tone/Ball_Spray_Brush_4' p=None>, <ManagedBrush u'tone/Twisted_Hair_Brush' p=None>, <ManagedBrush u'tone/SixSax_Hair' p=None>, <ManagedBrush u'tone/Round_Hairy' p=None>, <ManagedBrush u'tone/Hairy_Brush' p=None>, <ManagedBrush u'tone/Bristle_Brush_1' p=None>, <ManagedBrush u'tone/Bristle_Brush_2' p=None>, <ManagedBrush u'tone/Bristle_Brush_3' p=None>, <ManagedBrush u'tone/Bush_Brush' p=None>, <ManagedBrush u'tone/Oil_Brush_1' p=None>, <ManagedBrush u'tone/Oil_Brush_2' p=None>, <ManagedBrush u'tone/Wet_Oil_Brush_1' p=None>, <ManagedBrush u'tone/Wet_Oil_Flexible_Brush_1' p=None>, <ManagedBrush u'tone/Wet_Oil_Flexible_Brush_2' p=None>, <ManagedBrush u'tone/Wet_Oil_Flexible_Brush_3' p=None>, <ManagedBrush u'tone/Oilpaint_1' p=None>, <ManagedBrush u'tone/Oilpaint_2' p=None>, <ManagedBrush u'tone/Oilpaint_3' p=None>, <ManagedBrush u'tone/Oilpaint_4' p=None>, <ManagedBrush u'tone/Oilpaint_5' p=None>, <ManagedBrush u'tone/Grass_Hairy_Oil' p=None>, <ManagedBrush u'tone/Bristle_Blender_1' p=None>, <ManagedBrush u'tone/Bristle_Blender_2' p=None>, <ManagedBrush u'tone/Oil_Pastel_Smudge' p=None>, <ManagedBrush u'tone/Oil_Pastel_1' p=None>, <ManagedBrush u'tone/Oil_Pastel_2' p=None>, <ManagedBrush u'tone/Wax_Crayon' p=None>, <ManagedBrush u'tone/Round_Tint_1' p=None>, <ManagedBrush u'tone/Round_Tint_2' p=None>, <ManagedBrush u'tone/Hard_Round_Tint' p=None>, <ManagedBrush u'tone/Paint_Smudge_H1' p=None>, <ManagedBrush u'tone/Paint_Smudge_H2' p=None>, <ManagedBrush u'tone/Paint_Smudge_H3' p=None>, <ManagedBrush u'tone/Paint_Smudge_S1' p=None>, <ManagedBrush u'tone/Splatter_Smudge_1' p=None>, <ManagedBrush u'tone/Splatter_Smudge_2' p=None>, <ManagedBrush u'tone/Splatter_Smudge_3' p=None>, <ManagedBrush u'tone/Big_Blender' p=None>, <ManagedBrush u'tone/Basic_Smudge' p=None>, <ManagedBrush u'tone/Round_Hard_Blender' p=None>, <ManagedBrush u'tone/OpenCanvas_Soft' p=None>, <ManagedBrush u'tone/OpenCanvas_Hard' p=None>, <ManagedBrush u'tone/Airbrush_Round' p=None>, <ManagedBrush u'tone/Glow_or_Airbrush' p=None>, <ManagedBrush u'tone/Felt_Pen_1' p=None>, <ManagedBrush u'tone/Wet_Felt_Pen' p=None>, <ManagedBrush u'tone/Wet_Felt_Pen_Light' p=None>, <ManagedBrush u'tone/Moist_Colorpencil_1' p=None>, <ManagedBrush u'tone/Moist_Colorpencil_2' p=None>, <ManagedBrush u'tone/Wet_Colorpencil' p=None>, <ManagedBrush u'tone/Watercolor_Speed_Eraser' p=None>, <ManagedBrush u'tone/WaterC_Round_J1' p=None>, <ManagedBrush u'tone/WaterC_Round_J2' p=None>, <ManagedBrush u'tone/WaterC_Round_J3' p=None>, <ManagedBrush u'tone/WaterC_Round_J4' p=None>, <ManagedBrush u'tone/WaterC_Round_J1_Inv' p=None>, <ManagedBrush u'tone/WaterC_Round_J2_Inv' p=None>, <ManagedBrush u'tone/WaterC_Round_J3_Inv' p=None>, <ManagedBrush u'tone/Watercolor_Round_1' p=None>, <ManagedBrush u'tone/Watercolor_Round_2' p=None>, <ManagedBrush u'tone/Watercolor_Knife_1' p=None>, <ManagedBrush u'tone/Watercolor_Knife_2' p=None>, <ManagedBrush u'tone/Splash_Watercolor_Brush_1' p=None>, <ManagedBrush u'tone/Splash_Watercolor_Brush_2' p=None>, <ManagedBrush u'tone/Watery_Spots_1' p=None>], u'classic': [<ManagedBrush u'classic/pencil' p=None>, <ManagedBrush u'classic/charcoal' p=None>, <ManagedBrush u'classic/dry_brush' p=None>, <ManagedBrush u'classic/rounded' p=None>, <ManagedBrush u'classic/knife' p=None>, <ManagedBrush u'classic/smudge+paint' p=None>, <ManagedBrush u'classic/ink_eraser' p=None>, <ManagedBrush u'classic/blend+paint' p=None>, <ManagedBrush u'classic/modelling2' p=None>, <ManagedBrush u'classic/modelling' p=None>, <ManagedBrush u'classic/marker_fat' p=None>, <ManagedBrush u'classic/marker_small' p=None>, <ManagedBrush u'classic/kabura' p=None>, <ManagedBrush u'classic/pen' p=None>, <ManagedBrush u'classic/slow_ink' p=None>, <ManagedBrush u'classic/pointy_ink' p=None>, <ManagedBrush u'classic/brush' p=None>, <ManagedBrush u'classic/textured_ink' p=None>, <ManagedBrush u'classic/ink_blot' p=None>, <ManagedBrush u'classic/coarse_bulk_3' p=None>, <ManagedBrush u'classic/coarse_bulk_2' p=None>, <ManagedBrush u'classic/coarse_bulk_1' p=None>, <ManagedBrush u'classic/short_grass' p=None>, <ManagedBrush u'classic/long_grass' p=None>, <ManagedBrush u'classic/bulk' p=None>, <ManagedBrush u'classic/calligraphy' p=None>, <ManagedBrush u'classic/puantilism' p=None>, <ManagedBrush u'classic/puantilism2' p=None>, <ManagedBrush u'classic/impressionism' p=None>, <ManagedBrush u'classic/imp_details' p=None>, <ManagedBrush u'classic/imp_blending' p=None>, <ManagedBrush u'classic/wet_knife' p=None>, <ManagedBrush u'classic/blending_knife' p=None>, <ManagedBrush u'classic/smudge' p=None>, <ManagedBrush u'classic/blur' p=None>], u'deleted': [<ManagedBrush u'ramon/B-pencil' p=None>, <ManagedBrush u'ramon/2B_pencil' p=None>, <ManagedBrush u'ramon/P._Shade' p=None>, <ManagedBrush u'ramon/Sketch_1' p=None>, <ManagedBrush u'ramon/Soft_Eraser' p=None>, <ManagedBrush u'ramon/Hard_Eraser' p=None>, <ManagedBrush u'ramon/Thin_Pen' p=None>, <ManagedBrush u'ramon/Pen' p=None>, <ManagedBrush u'ramon/PenBrush' p=None>, <ManagedBrush u'ramon/Marker' p=None>, <ManagedBrush u'ramon/Delayed_' p=None>, <ManagedBrush u'ramon/100%_Opaque' p=None>, <ManagedBrush u'ramon/Round' p=None>, <ManagedBrush u'ramon/Round_Bl' p=None>, <ManagedBrush u'ramon/BigAirbrush' p=None>, <ManagedBrush u'ramon/RS_blendOP' p=None>, <ManagedBrush u'ramon/wet_round' p=None>, <ManagedBrush u'ramon/Wet_Direction' p=None>, <ManagedBrush u'ramon/Classic_Paint' p=None>, <ManagedBrush u'ramon/Blur_Fast' p=None>, <ManagedBrush u'ramon/Smear' p=None>, <ManagedBrush u'ramon/Knife' p=None>, <ManagedBrush u'ramon/Grain' p=None>, <ManagedBrush u'ramon/Dirty_Noise' p=None>, <ManagedBrush u'ramon/Pastel_1' p=None>, <ManagedBrush u'ramon/Beamlight' p=None>, <ManagedBrush u'ramon/Glow_Airbrush' p=None>, <ManagedBrush u'ramon/Clouds' p=None>, <ManagedBrush u'tone/WaterC_Round_J3copy' p=None>, <ManagedBrush u'deevad/kneaded_erasercopy' p=None>, <ManagedBrush u'deevad/fillcopy' p=None>, <ManagedBrush u'deevad/large_hard_erasercopy' p=None>], u'set#2': [<ManagedBrush u'deevad/rough' p=None>, <ManagedBrush u'deevad/4H_pencil' p=None>, <ManagedBrush u'deevad/2B_pencil' p=None>, <ManagedBrush u'deevad/pen' p=None>, <ManagedBrush u'deevad/kneaded_eraser' p=None>, <ManagedBrush u'deevad/kneaded_eraser_large' p=None>, <ManagedBrush u'deevad/large_hard_eraser' p=None>, <ManagedBrush u'deevad/thin_hard_eraser' p=None>, <ManagedBrush u'deevad/basic_digital_brush' p=None>, <ManagedBrush u'deevad/basic_digital_brush_smudging' p=None>, <ManagedBrush u'deevad/basic_digital_knife' p=None>, <ManagedBrush u'deevad/basic_digital_knife_smudging' p=None>, <ManagedBrush u'deevad/detail_brush_large' p=None>, <ManagedBrush u'deevad/detail_brush_large_glazing' p=None>, <ManagedBrush u'deevad/detail_brush_thin' p=None>, <ManagedBrush u'deevad/detail_brush_thin_glazing' p=None>, <ManagedBrush u'deevad/rigger_brush' p=None>, <ManagedBrush u'deevad/rigger_brush_thin' p=None>, <ManagedBrush u'deevad/thin_watercolor' p=None>, <ManagedBrush u'deevad/large_watercolor_fringe' p=None>, <ManagedBrush u'deevad/watercolor_glazing' p=None>, <ManagedBrush u'deevad/watercolor_expressive' p=None>, <ManagedBrush u'deevad/only_water_fringe' p=None>, <ManagedBrush u'deevad/sponge_smudging' p=None>, <ManagedBrush u'deevad/ballpen' p=None>, <ManagedBrush u'deevad/pen-note' p=None>, <ManagedBrush u'deevad/soft-dip-pen' p=None>, <ManagedBrush u'deevad/liner' p=None>, <ManagedBrush u'deevad/brush' p=None>, <ManagedBrush u'deevad/fill' p=None>, <ManagedBrush u'deevad/spray' p=None>, <ManagedBrush u'deevad/spray2' p=None>, <ManagedBrush u'deevad/chalk' p=None>, <ManagedBrush u'deevad/grainy_blending' p=None>, <ManagedBrush u'deevad/blending' p=None>, <ManagedBrush u'deevad/airbrush' p=None>], u'set#4': [<ManagedBrush u'kaerhon_v1/flat_bar_l' p=None>, <ManagedBrush u'kaerhon_v1/inkster_l' p=None>, <ManagedBrush u'kaerhon_v1/extreme_round_l' p=None>, <ManagedBrush u'kaerhon_v1/ink_slow_s' p=None>, <ManagedBrush u'kaerhon_v1/ink-slowline_s' p=None>, <ManagedBrush u'kaerhon_v1/smudge_ink(0.7)_sm' p=None>, <ManagedBrush u'kaerhon_v1/paint_radius_2_sm' p=None>, <ManagedBrush u'kaerhon_v1/Smear_sm' p=None>, <ManagedBrush u'kaerhon_v1/paint_sm' p=None>, <ManagedBrush u'kaerhon_v1/paint_barrr_sm' p=None>, <ManagedBrush u'kaerhon_v1/wet_paint_sm' p=None>, <ManagedBrush u'kaerhon_v1/Dirty_Transparent_sk' p=None>, <ManagedBrush u'kaerhon_v1/classic_sk' p=None>, <ManagedBrush u'kaerhon_v1/airsmudgeultimate_sk' p=None>, <ManagedBrush u'kaerhon_v1/Sketcher2_sk' p=None>, <ManagedBrush u'kaerhon_v1/airsmudge_a' p=None>, <ManagedBrush u'kaerhon_v1/airbruch_press_a' p=None>, <ManagedBrush u'kaerhon_v1/Airbrush_a' p=None>, <ManagedBrush u'kaerhon_v1/classicroundblock_static_c' p=None>, <ManagedBrush u'kaerhon_v1/fill_c' p=None>], u'Offset-AnTi': [<ManagedBrush u'Anti/Offset-XY' p=None>, <ManagedBrush u'Anti/Offset-Angular1' p=None>, <ManagedBrush u'Anti/Offset-Angular2' p=None>, <ManagedBrush u'Anti/Off-Rake1' p=None>, <ManagedBrush u'Anti/Off-Rake2' p=None>, <ManagedBrush u'Anti/Off-Rake3' p=None>, <ManagedBrush u'Anti/Off-Bleed' p=None>, <ManagedBrush u'Anti/Off-Dripping' p=None>, <ManagedBrush u'Anti/Off-Outline' p=None>, <ManagedBrush u'Anti/Off-V' p=None>, <ManagedBrush u'Anti/Off-Arcs' p=None>, <ManagedBrush u'Anti/Off-Carneval' p=None>, <ManagedBrush u'Anti/Off-Flash' p=None>, <ManagedBrush u'Anti/Off-Weave' p=None>, <ManagedBrush u'Anti/Off-Stitch' p=None>, <ManagedBrush u'Anti/Off-Lines' p=None>, <ManagedBrush u'Anti/Off-Dots' p=None>, <ManagedBrush u'Anti/Off-Bands' p=None>, <ManagedBrush u'Anti/Off-ZigZag' p=None>, <ManagedBrush u'Anti/Off-Sawtooth' p=None>, <ManagedBrush u'Anti/Off-Tube' p=None>, <ManagedBrush u'Anti/Off-Wave' p=None>, <ManagedBrush u'Anti/Off-Tiled' p=None>, <ManagedBrush u'Anti/Off-Squared' p=None>, <ManagedBrush u'Anti/Off-Squiggle' p=None>, <ManagedBrush u'Anti/Off-Chaos' p=None>, <ManagedBrush u'Anti/Off-Strokes' p=None>, <ManagedBrush u'Anti/Off-Spiral' p=None>, <ManagedBrush u'Anti/Off-Feather' p=None>, <ManagedBrush u'Anti/Off-Dots2' p=None>], u'experimental': [<ManagedBrush u'experimental/1pixel' p=None>, <ManagedBrush u'experimental/pixel_hardink' p=None>, <ManagedBrush u'experimental/pixelblocking' p=None>, <ManagedBrush u'experimental/basic' p=None>, <ManagedBrush u'experimental/subtle_pencil' p=None>, <ManagedBrush u'experimental/hard_sting' p=None>, <ManagedBrush u'experimental/glow' p=None>, <ManagedBrush u'experimental/soft' p=None>, <ManagedBrush u'experimental/track' p=None>, <ManagedBrush u'experimental/speed_blot' p=None>, <ManagedBrush u'experimental/fur' p=None>, <ManagedBrush u'experimental/soft_irregular' p=None>, <ManagedBrush u'experimental/irregular_ink' p=None>, <ManagedBrush u'experimental/bubble' p=None>, <ManagedBrush u'experimental/small_blot' p=None>, <ManagedBrush u'experimental/particules_3' p=None>, <ManagedBrush u'experimental/spaced-blot' p=None>, <ManagedBrush u'experimental/hard_blot' p=None>, <ManagedBrush u'experimental/sewing' p=None>, <ManagedBrush u'experimental/particules_eraser' p=None>, <ManagedBrush u'experimental/pick_and_drag' p=None>, <ManagedBrush u'experimental/leaves' p=None>, <ManagedBrush u'experimental/DNA_brush' p=None>], u'set#3': [<ManagedBrush u'tanda/acrylic-03-only-water' p=None>, <ManagedBrush u'tanda/acrylic-03-paint' p=None>, <ManagedBrush u'tanda/acrylic-03-with-water' p=None>, <ManagedBrush u'tanda/acrylic-04-only-water' p=None>, <ManagedBrush u'tanda/acrylic-04-paint' p=None>, <ManagedBrush u'tanda/acrylic-04-with-water' p=None>, <ManagedBrush u'tanda/acrylic-05-only-water' p=None>, <ManagedBrush u'tanda/acrylic-05-paint' p=None>, <ManagedBrush u'tanda/acrylic-05-with-water' p=None>, <ManagedBrush u'tanda/charcoal-01' p=None>, <ManagedBrush u'tanda/charcoal-03' p=None>, <ManagedBrush u'tanda/charcoal-04' p=None>, <ManagedBrush u'tanda/charcoal-blur1' p=None>, <ManagedBrush u'tanda/marker-01' p=None>, <ManagedBrush u'tanda/marker-05' p=None>, <ManagedBrush u'tanda/oil-01-clean' p=None>, <ManagedBrush u'tanda/oil-01-paint' p=None>, <ManagedBrush u'tanda/oil-03-clean' p=None>, <ManagedBrush u'tanda/oil-03-paint' p=None>, <ManagedBrush u'tanda/oil-06-clean' p=None>, <ManagedBrush u'tanda/oil-06-paint' p=None>, <ManagedBrush u'tanda/oil-mop' p=None>, <ManagedBrush u'tanda/pencil-2b' p=None>, <ManagedBrush u'tanda/pencil-8b' p=None>, <ManagedBrush u'tanda/splatter-02' p=None>, <ManagedBrush u'tanda/splatter-04' p=None>, <ManagedBrush u'tanda/texture-03' p=None>, <ManagedBrush u'tanda/texture-06' p=None>, <ManagedBrush u'tanda/texture-12' p=None>, <ManagedBrush u'tanda/water-01' p=None>, <ManagedBrush u'tanda/water-02' p=None>, <ManagedBrush u'tanda/water-05' p=None>, <ManagedBrush u'tanda/water-06' p=None>, <ManagedBrush u'tanda/watercolor-02-paint' p=None>, <ManagedBrush u'tanda/watercolor-02-water' p=None>]}), 'faves': (None, []), 'brushmanager.FAVORITES_BRUSH_GROUP': ('global', u'favorites')}
KeyError: u'favorites'
``````

```
```

**Comment by lunlune** [comment]
oh... i resolved it with deleting the settings files.
so, sorry for bothering.

**Comment by achadwick** [comment]
Thanks for the report though :smile: 

Technically it's a duplicate of #641, already fixed by @danpla's awesomeness. Sadly the fix isn't on the radar for 1.2.1. It would be nice to have a good workaround in the release notes, however.

I'll treat this as a docs task for 1.2.1.

**Comment by achadwick** [comment]
Drafting now, and it's in the notes ready for the release.
